appsignals-tests/contract-tests/src/test/java/software/amazon/opentelemetry/appsignals/test/jdbc/operationtests/JdbcCreateDatabaseOperationTester.java

```diff
@@ -0,0 +1,27 @@
+package software.amazon.opentelemetry.appsignals.test.jdbc.operationtests;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import software.amazon.opentelemetry.appsignals.test.utils.ResourceScopeSpan;
+import software.amazon.opentelemetry.appsignals.test.utils.SemanticConventionsConstants;
+
+import static software.amazon.opentelemetry.appsignals.test.jdbc.operationtests.DBOperation.CREATE_DATABASE;
+
+public class JdbcCreateDatabaseOperationTester extends JdbcOperationTester {
+
+    public JdbcCreateDatabaseOperationTester(String dbSystem, String dbUser, String jdbcUrl, String dbName, String dbTable) {
+        super(dbSystem, dbUser, jdbcUrl, dbName, CREATE_DATABASE.toString(), dbTable);
+    }
+
+    @Override
+    protected void assertOperationSemanticConventions(ResourceScopeSpan resourceScopeSpan) {
```

Comment:
in the future we can also add decorators so per engine per operation we can have different assertion behavior.